src/main/java/com/intellij/compiler/notNullVerification/ImplicitThrowOnNullMethodVisitor.java

```diff
@@ -30,8 +30,8 @@
 
     private final Set<String> nullableAnnotations;
 
-    ImplicitThrowOnNullMethodVisitor(@Nullable final MethodVisitor methodVisitor, @NotNull final Type[] argumentTypes, @NotNull final Type returnType, final int access, @NotNull final String methodName, @NotNull final ClassInfo classInfo, @NotNull final Set<String> nullableAnnotations, @Nullable final Boolean isAnonymousClass) {
-        super(AsmUtils.ASM_OPCODES_VERSION, methodVisitor, argumentTypes, returnType, access, methodName, classInfo, true, isAnonymousClass);
+    ImplicitThrowOnNullMethodVisitor(@Nullable final MethodVisitor methodVisitor, @NotNull final Type[] argumentTypes, @NotNull final Type returnType, final int access, @NotNull final String methodName, @NotNull final ClassInfo classInfo, @NotNull final Set<String> nullableAnnotations, @Nullable final Boolean isAnonymousClass, final boolean warnInsteadOfThrow) {
```

Comment:
`Boolean isAnonymous` as an Object type vs `boolean warnInsteadOfThrow` is primitive.